Dockerfile

```
@@ -1,16 +1,21 @@
-FROM erlang:22.3.2-alpine as node-builder
-RUN apk add --no-cache --update \
-    git tar build-base linux-headers autoconf automake libtool pkgconfig \
-    dbus-dev bzip2 bison flex gmp-dev cmake lz4 libsodium-dev openssl-dev \
-    sed wget curl
+FROM ubuntu:18.04 as node-builder
 
+ENV DEBIAN_FRONTEND noninteractive
+ENV PATH /root/.cargo/bin:${PATH}
 ENV CC=gcc CXX=g++ CFLAGS="-U__sun__" \
     ERLANG_ROCKSDB_OPTS="-DWITH_BUNDLE_SNAPPY=ON -DWITH_BUNDLE_LZ4=ON" \
     ERL_COMPILER_OPTIONS="[deterministic]" \
     PATH="/root/.cargo/bin:$PATH" \
     RUSTFLAGS="-C target-feature=-crt-static"
 
-# install Rust toolchain
+# install erlang
+RUN apt-get update && apt-get install -y libsodium-dev gcc
+RUN apt-get install -y libssl-dev build-essential curl make g++ git gnupg2 wget cmake
```

```suggestion
RUN apt-get update && apt-get install -y build-essential cmake autoconf automake bison flex libtool libsodium-dev libssl-dev libdbus-1-dev libgmp-dev libbz2-dev liblz4-dev
```

This is the list i use on an ubuntu builder